newIDE/app/src/EventsFunctionsExtensionsLoader/Storage/BrowserEventsFunctionsExtensionWriter.js

```diff
@@ -0,0 +1,70 @@
+// @flow
+import { serializeToJSObject } from '../../Utils/Serializer';
+
+const downloadStringContentAsFile = (
+  filename: string,
+  content: string
+): Promise<void> => {
```

Comment:
here and below, consider using `async` functions so you can avoid Promise.resolve (just return) or Promise.reject (just throw)